actionview/test/template/render_test.rb

```diff
@@ -309,6 +309,14 @@ def test_render_partial_with_nil_collection_should_return_nil
     assert_nil @view.render(:partial => "test/customer", :collection => nil)
   end
 
+  def test_render_partial_collection_for_non_array
+    customers = Enumerator.new do |y|
+      y.yield(Customer.new("david"))
+      y.yield(Customer.new("mary"))
+    end
+    assert_equal "Hello: davidHello: mary", @view.render(:partial => "test/customer", :collection => customers)
```

Comment:
Ah, very good. I knew >= 1.9 was the preferred syntax generally, but thought you were asking to follow the style used in the rest of the file.

Just pushed a commit to change over to `key: :value`.